### PR TITLE
Add 'activate_recruiter_on_start' config var

### PR DESCRIPTION
**Pull request by fmhoeger**
* Added new `activate_recruiter_on_start` config variable which defaults to `True`. If the variable is set to `True`, then existing behavior is preserved; if set to `False`, then `open_recruitment()` is skipped when the experiment launches. This means that the user has to manually initialize recruitment (e.g. via the Prolific panel).

### CODE REVIEW

**Comment by pmcharrison** [comment]
Thanks @fmhoeger, the new content looks very good. Would you be happy to add a test? I think it would be valuable here and quite easy based on existing Dallinger tests.

**Comment by fmhoeger** [comment]
@pmcharrison Added tests.
Ready for review.

File: tests/test_experiment_server.py [hunk 1]
```diff
@@ -1568,6 +1568,35 @@ def test_launch(self, webapp):
...
+            mock_exp.recruiter.open_recruitment.return_value = {
+                "items": ["item"],
+                "message": "a message",
+            }
+            mock_class.return_value = mock_exp
+            resp = webapp.post("/launch", data={})
+        assert resp.status_code == 200
+        mock_exp.recruiter.open_recruitment.assert_called()
+
+    def test_launch_without_recuitment(self, webapp, active_config):
```
Comment (pmcharrison) [inline tests/test_experiment_server.py:1587]: Typo

File: tests/test_experiment_server.py [hunk 2]
```diff
@@ -1568,6 +1568,35 @@ def test_launch(self, webapp):
         data = json.loads(resp.get_data())
         assert "recruitment_msg" in data
 
+    def test_launch_with_recuitment(self, webapp, active_config):
```
Comment (pmcharrison) [inline tests/test_experiment_server.py:1571]: Typo

**Review by pmcharrison — COMMENTED** [review]
A couple of typos but otherwise looks good!

File: tests/test_experiment_server.py (hunk 1 above)
Comment (fmhoeger) [reply tests/test_experiment_server.py]: Fixed.

File: tests/test_experiment_server.py (hunk 2 above)
Comment (fmhoeger) [reply tests/test_experiment_server.py]: Fixed.

**Review by pmcharrison — APPROVED** [review]

**Pull request merged**